### DISCUSSION TITLE
[委托]霉运霉运快走开!!

### DISCUSSION THREAD

**Discussion by Dongxiang233** [share121/inter-knot#674] [其他]
我好像被芭菜大厦里面的黄金邦布坦咒了，最近财运好差。
据说只要找几个凶神恶煞的家伙揍那个邦布一顿，让它怕了，它就会认怂解除组咒
所以…有没有人能替我进空洞揍它一顿?
拜托拜托，救救我啊啊啊!
![Uploading 9fdc8ee0079a684da3750af7bda6212.png…]()

**Comment by NIUTE-SKMD** [comment]
这个委托我接了

**Comment by Elysia-Ego-093** [comment]
这个委托我接了！

**Comment by Lain0100** [comment]
这个委托我接了！

**Comment by wushanef** [comment]
小智？你怎么在这，快回来排练，mmk快生气了！

**Comment by feco-y** [comment]
《坦咒》《组咒》

**Comment by aaa00073** [comment]
你好，我先接下委托去找它，然后从它那里得到双倍报酬再回来，只要你再给我4倍报酬，我就再去找它

**Comment by infinite-vector** [comment]
这个委托我接了！
